realtime/connection.py

```diff
@@ -156,4 +156,4 @@ def summary(self) -> None:
         """
         for topic, chans in self.channels.items():
             for chan in chans:
-                print(f"Topic: {topic} | Events: {[e for e, _ in chan.callbacks]}]")
+                print(f"Topic: {topic} | Events: {[e for e, _ in chan.listeners]}]")
```

Comment:
This bug fix is not really related to the feature request. But I simply did it, while working on it. If you want, I can create a new bug ticket and solve it there.